extensions/ql-vscode/src/common/app.ts

```diff
@@ -4,6 +4,11 @@ import { AppEventEmitter } from "./events";
 import { Logger } from "./logging";
 import { Memento } from "./memento";
 import { AppCommandManager } from "./commands";
+import type {
```

Comment:
Since this is not a massively well known feature of TS, it might be worth putting a comment to explain what is going on. What do you think? I'm happy either way.

Reply:
You're right it's not a hugely well known feature of TypeScript. I didn't know about it until about 6 months ago. But I don't think commenting in the code is a helpful way of teaching people, because they may not see this particular copy and I don't want to include a comment on every single usage. There are 32 other examples already and my hope would be that we add more because it's a cool and useful feature. So I hope that over time the need for explanation will be minimal as everyone will see this feature enough to know what it does.